app/presenters/waste_carriers_engine/registration_presenter.rb

```diff
@@ -0,0 +1,64 @@
+module WasteCarriersEngine
+  class RegistrationPresenter < BasePresenter
```

Comment:
I'm not sure about naming this the RegistrationPresenter – it's only used for certificates at the moment but that name makes it sound like its scope is a lot bigger. If we're intending to start using this logic elsewhere in the engine then I'm fine with it.